ui/conftest.py

```diff
@@ -24,3 +24,14 @@ def mock_moira(mocker):
 def mock_moira_client(mocker):
     """Return a fake moira client"""
     return mocker.patch('ui.utils.get_moira_client', autospec=True)
+
+
+@pytest.fixture(autouse=True)
+def youtube_mock(mocker):
+    """
+    Mocks calls for youtube api tests
+    """
+    mocker.patch('cloudsync.youtube.boto3')
```

Comment:
It looks like we frequently patch these methods elsewhere. Can we yield these patched values and use this fixture to do the mocking in `youtube_test.py` and `tasks_test.py`?

Reply:
I have a similar fixture (`youtube_mock`) in `cloudsync/conftest.py  that's used in `youtube_test.py` and `tasks_test.py`.  Do you mean use one fixture for both `ui` and `cloudsync` tests that need it?

Reply:
Yes